app/Http/Controllers/UserController.php

```diff
@@ -3,17 +3,65 @@
 namespace App\Http\Controllers;
 
 use App\User;
+use App\Category;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
 
 class UserController extends Controller
 {
     public function dashboardView()
     {
         $learnedWordsNumber = Auth::user()->learnedWord->count();
         $learnedLessonsNumber = Auth::user()->learnedLesson->count();
+
+        $activities = collect();
```

Comment:
I think in ER Diagram there is Acitivity diagram, when you have Activity schema you don't need to manually code the activity logic, you just have to query them

Reply:
Actually, I didn't make Activity table because the data of Activity table can be made from 
data of Relationship and Lesson. If this change isn't good, I'm going to make the table and use it.

Reply:
@KoutarouTakayama but then again, from the ER diagram that is given there is activity table, so we should follow that one

Reply:
i understood. I make the table and use it.

Reply:
Please apply the Polymorphic Relationship In Laravel